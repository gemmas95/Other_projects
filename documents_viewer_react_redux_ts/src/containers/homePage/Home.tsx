import React from "react";

const Home: React.FC = () => {
  return (
    <>
      <h1>Welcome to Documents viewer!</h1>
      <p>
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Veniam
        mollitia quaerat tenetur ratione numquam, accusantium nam eveniet sed
        dolore voluptate nulla, consectetur quidem ducimus ea unde beatae
        facilis perferendis excepturi!
      </p>
    </>
  );
};

export default Home;
