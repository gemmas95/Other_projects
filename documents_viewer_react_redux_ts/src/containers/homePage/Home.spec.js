import { render } from "@testing-library/react";
import React from "react";
import { BrowserRouter } from "react-router-dom";
import Home from "./Home";

describe("Home", () => {
  it("should match snapshot", () => {
    const HomeTree = render(
      <BrowserRouter>
        <Home />
      </BrowserRouter>
    );
    expect(HomeTree).toMatchSnapshot();
  });
});
