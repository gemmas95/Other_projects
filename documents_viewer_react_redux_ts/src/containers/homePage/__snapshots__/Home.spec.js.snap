// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1>
        Welcome to Documents viewer!
      </h1>
      <p>
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Veniam mollitia quaerat tenetur ratione numquam, accusantium nam eveniet sed dolore voluptate nulla, consectetur quidem ducimus ea unde beatae facilis perferendis excepturi!
      </p>
    </div>
  </body>,
  "container": <div>
    <h1>
      Welcome to Documents viewer!
    </h1>
    <p>
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Veniam mollitia quaerat tenetur ratione numquam, accusantium nam eveniet sed dolore voluptate nulla, consectetur quidem ducimus ea unde beatae facilis perferendis excepturi!
    </p>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
